app/views/usage/_total_usage.html.erb

```diff
@@ -31,7 +31,7 @@
       <%= select_tag "csv-field-sep", options_for_select(separators, separators[0]), {class: "single-char-select"} %>
     </form>
   </div>
-  <% if current_user.can_super_admin? %>
+  <% if current_user.can_super_admin? || current_user.can_org_admin? %>
```

Comment:
I think this line would be granting access to organizational administrators to all of DMP Assistant statistics. This is something that we want to look at in the future but for the time being, org admins should only be able to see their organizational stats.